frame/support/procedural/src/pallet/expand/genesis_config.rs

```diff
@@ -126,7 +135,7 @@ pub fn expand_genesis_config(def: &mut Def) -> proc_macro2::TokenStream {
 						stringify!($pallet_name),
 						"` does not have the std feature enabled, this will cause the `",
 						$pallet_path,
-						"::GenesisConfig` type to be undefined."
+						"::GenesisConfig` type to not implement serde traits."
```

Comment:
When std feature is not enabled for pallet, the GenesisConfig will be defined, but `serde::{Serialize,Deserialize}` traits will not be implemented.
    
I've decided to change this compiler error to indicate the reason of latter errors occuring during compilation.
    
This is actually temporary and serde traits will be enabled with together with `serde` support in frame.